python/ray/serve/config.py

```diff
@@ -39,7 +40,12 @@ class AutoscalingConfig(BaseModel):
     initial_replicas: Optional[NonNegativeInt] = None
     max_replicas: PositiveInt = 1
 
-    target_num_ongoing_requests_per_replica: PositiveFloat = 1.0
+    # DEPRECATED: replaced by target_ongoing_requests
+    target_num_ongoing_requests_per_replica: PositiveFloat = Field(
+        default=1.0,
+        description="[DEPRECATED] Please use `target_ongoing_requests` instead.",
```

Comment:
Let's make sure to add a warning for this as well please

Reply:
Yup sorry forgot about the warnings. Added warnings in `api.py` and `deployment.py`.